controllers/Role.js

```diff
@@ -0,0 +1,121 @@
+//sImport Role Model
+import db from "../config/db.config.js";
+import Role from "../models/Role.js";
+
+ // Get all Role
+export const getRoles = async (req, res) => {
+    const name = req.query.name;
+    const description=req.query.description;
```

Comment:
@mukeshsin why do you need 
const name = req.query.name;
    const description=req.query.description;
 these here, you are not using them.

Reply:
because there are not sent any data request  we are requesting to get all role by single query. i know i should use const name=req.body.name. as well as  same description. ok sorry i will remember. promise

Reply:
@meanstack9716